Misc/largestRange.java

```diff
@@ -0,0 +1,46 @@
+import java.util.*;
+
+public class largestRange {
+
+  // Finds the length of longest occurring consecutive numbers range in an array
+  public static int longestRange(int[] nums) {
+    int longestRange = 0;
+    HashMap<Integer, Boolean> num = new HashMap<>();
+
+    /**
+     * Stores a mapping of a number to whether the current number is part of a particular
+     * consecutive range or not.
+     */
+    for (int x : nums) num.put(x, true);
```

Comment:
I'm not sure but can't we just use lookahead type of approach over here? 

For example, 

```java
int[] input_arr = new int[]{0,1,2,4,5,6,0,1};
int count = 1;
int temp_count = 1;
for (int i = 1; i < input_arr.length; i++) {
    if (input_arr[i] != input_arr[i - 1] + 1) {
        if (temp_count > count) {
            count = temp_count;
        }
        temp_count = 1;
    } else temp_count++;
}
System.out.println(count); // Prints 3
```

If I have misunderstood requirements or this does not work as intended (I haven't tested it, so it is be possible), please let me know. 

If above works just as fine, please can you explain this approach's necessity? 

Please do needful and let me know.